lib/core/lib/ops/teamOps.js

```diff
@@ -286,7 +286,7 @@ function createDefaultUser (job, next) {
   const { organizationId } = job.params
 
   userOps.insertUser(job.client, { id, name, organizationId }, (err, user) => {
-    if (err) return next(Boom.badImplementation(err))
+    if (err) return next(err)
```

Comment:
The file has code that wraps the err in Boom in several places, which seems not so good as the err can be wrapped in lower level code. All error wrapping and passing in Udaru needs some rigorous review and change.

But also the proposed change is not good as it might escape unwrapped errs.

The PR is ok excepting this change.